compliance/src/main/java/bio/terra/pearl/compliance/SyncVantaUsers.java

```diff
@@ -208,25 +211,17 @@ private <T extends VantaObject> Collection<T> collectAllData(WebClient wc, Param
         AtomicInteger cursorCount = new AtomicInteger(0);
 
         while (cursorCount.get() == 0 || results.get().getPageInfo().isHasNextPage()) {
-            log.debug("Getting cursor number {}", cursorCount);
+            log.info("Getting {} cursor number {}", type.getType().getTypeName(), cursorCount);
```

Comment:
More often than not, it's helpful to see the progress through the cursors in the logs.  So this will be `info`.